### ISSUE TITLE
sfpowerkit:pool:fetch command no longer returns the authURL

### ISSUE THREAD

**Issue by 1ktribble** [dxatscale/sfpowerkit#721]
**Describe the bug**
As of the latest release, [May 22-2](https://github.com/dxatscale/sfpowerkit/releases/tag/v4.2.13), I'm unable to retrieve the sfdxAuthUrl attribute from the `sfdx sfpowerkit:pool:fetch` command. I'm logged into my dev hub using an auth url, and this action was working until this morning. All other attributes are retrieved successfully.

e.g.
```json
{
  "status": 0,
  "result": {
    "Id": "2SR3b-------------",
    "orgId": "00D7e----------",
    "loginURL": "https://customization-drive-8431.my.salesforce.com/",
    "username": "test-kwhodjhqaeri@example.com",
    "password": "uk@i9dtlxztkR",
    "expiryDate": "2022-08-11",
    "sfdxAuthUrl": null,
    "status": "Assigned"
  }
}

```

**To Reproduce**

1. Log into Dev Hub using auth url.
2. Create a scratch org pool using `sfdx sfpowerkit:pool:create` command
3. Fetch a scratch org using `sfdx sfpowerkit:pool:fetch` command.

**Expected behavior**

` "sfdxAuthUrl": null,` should be replaced with an authUrl value.

**Screenshots**
<img width="1326" alt="Screen Shot 2022-08-11 at 9 08 16 AM" src="https://user-images.githubusercontent.com/7140983/184140593-6db93774-7b18-4c18-a68b-34f1b7933d3b.png">
<img width="855" alt="Screen Shot 2022-08-11 at 9 05 55 AM" src="https://user-images.githubusercontent.com/7140983/184140595-14d89e63-785c-4028-9718-f21da0041800.png">

**Desktop (please complete the following information):**

-   OS: Linux and MacOS
-   Node Version: 16.16.0
-   Version of sfpowerkit[e.g 1.45.0]: 4.2.13
-   Salesforce Org Version [ In case of pre release window ]: Summer 22

**Additional context**

**Comment by Caitlyn-Mills** [comment]
Hi @1ktribble thanks for raising. 
We did have this issue in 4.2.12, i have double checked it is resolved in 4.2.13. Can you please try creating a fresh pool? It's possible your fetch command has taken a scratch org that was created under 4.2.12

<img width="570" alt="image" src="https://user-images.githubusercontent.com/63215232/184259430-858c33d7-a020-48e0-89b3-4d47b5a52ae8.png">

Let me know if this solves the problem. 

Caitlyn

**Comment by 1ktribble** [comment]
I refreshed the pool, now it works as expected. Thanks!
